modules/Settings/Picklist/models/Module.php

```diff
@@ -424,4 +471,39 @@ public function clearPicklistCache(string $fieldName, string $moduleName)
 		\App\Cache::delete('getCloseStatesByName', \App\Module::getModuleId($moduleName));
 		\App\Cache::delete('getCloseStates', \App\Module::getModuleId($moduleName));
 	}
+
+	/**
+	 * Get all automation statuses.
+	 *
+	 * @return int[]
+	 */
+	public static function getAutomationStatus(): array
```

Comment:
```suggestion
	public static function getAutomationStatuses(): array
```

Reply:
English use of the word as a mass noun, as in considering their relative status.' So status and statuses can both be used as the plural, depending on the context, but status is more common